Pranjali/HTML/Assi6/assi6.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <style>
+            .a {
+                font-family: Arial, Helvetica, sans-serif;
+                background-color: rgb(137, 167, 224);
+                text-align: right;
+            }
+
+            .b {
+                background-color: rgb(137, 167, 224);
+                text-align: right;
+            }
+
+            #corners {
+                border-bottom-left-radius: 8px;
+                border-top-right-radius: 8px;
+            }
+        </style>
+    </head>
+    <body>
+        <h2 class="a" id="corners">MI-9 Intern HTML Assesment</h2>
+        <br />
+        <hr />
+        <h3 class="b" id="corners"><i>Personal Details</i></h3>
```

Comment:
use <em> tag instead <i> tag. beacause <i> tag is outdated. and also use <strong> tag instead of <b> tag